### PR TITLE
Fix critical codacy errors

### PR DESCRIPTION
**Pull request by Sophist-UK**
Part 1 of several PRs to fix codacy errors.

Most of these should be obviously transparent, but changes to meerk40t/core/drivers.py for `opt_jog_mode` almost certainly need double checking.

### CODE REVIEW

File: meerk40t/core/drivers.py [hunk 1]
```diff
@@ -191,7 +193,9 @@ def _fetch_next_item_from_spooler(self):
             self.spooled_item = element
         else:
             self.rapid = self.root_context.opt_rapid_between
-            self.jog = self.root_context.opt_jog_mode
+            # # there is a method called jog and the following line overwrites it
+            # # I can find no other references to self.jog as an int in the code.
+            # self.jog = self.root_context.opt_jog_mode
             try:
```
Comment (tatarize) [inline meerk40t/core/drivers.py:199]: This does nothing. Actually self.rapid also does nothing.

**Review by tatarize — APPROVED** [review]
Even if there were issues this is so chalked full of marked improvements that I'd take some issues which there shouldn't be any of.

**Comment by Sophist-UK** [comment]
Good to go.

**Review by tiger12506 — APPROVED** [review]
Tat already left a glowing review on this. The changes afterward seem minor and look ok to me.

**Pull request merged**